### ISSUE TITLE
Example: Multiply EC Point by Integer

### ISSUE THREAD

**Issue by wsw109** [nakov/Practical-Cryptography-for-Developers-Book#69]
maybe have a error here : "Example: Multiply EC Point by Integer":

![image](https://github.com/nakov/Practical-Cryptography-for-Developers-Book/assets/29395111/84244dc6-8dda-49f4-a187-b04cec771530)
----------
for  k = 6, G = {15,13},
P = k*G = 6*{15,13} = {90,78} mod 17 = {5,10}, which is not the {5,8}.

am I correct?

**Comment by kiankyars** [comment]
you are not correct, please look at the definition EC  scalar multiplication, it is not the same as normal multiplication
